packages/extension/.vscodeignore

```diff
@@ -0,0 +1,3 @@
+.vscode/**
+.vscode-test/**
+src/**
```

Comment:
How about explicitly only including what is needed? `package.json` and `README.md` are included automatically.
```suggestion
# Ignore everything
*
**/*
../**/*     # when `vsce` run with `--no-dependencies` this is not needed 
../../**/*  # when `vsce` run with `--no-dependencies` this is not needed 

# Add what we actually want
!out/extension/main.cjs
!out/extension/main.cjs.map
!language-configuration.json
!syntaxes/pli.merged.json
```